---
int32 goals_reached
int32 goals_cancelled